x-pack/plugins/observability_solution/logs_shared/public/components/logging/log_text_stream/loading_item_view.tsx

```diff
@@ -181,9 +188,15 @@ const ProgressCta: React.FC<ProgressCtaProps> = ({
 
   if (rangeEdge === 'now' && position === 'end') {
     return (
-      <EuiButton data-test-subj="infraProgressCtaStreamLiveButton" onClick={onStreamStart} size="s">
-        <FormattedMessage id="xpack.logsShared.logs.streamLive" defaultMessage="Stream live" />
-      </EuiButton>
+      <div role="cell">
```

Comment:
> [!TIP]
> `EuiButton` accepts the `role` attribute, do you think we can directly pass it there to not change the rendered markdown and avoid CSS side effects?

Reply:
I wanted to avoid changing the markup tbh but I followed the same [suggestion](https://github.com/elastic/observability-accessibility/issues/123) with the other button displayed at the top of the log stream. I think the idea was to have a wrapper with a `cell` role, and then a button as a child element with a `button` role by default.

<img width="901" alt="image" src="https://github.com/user-attachments/assets/ab92216c-6b9f-49b8-87e1-b07b482d33d6">